runtimes/pythonrt/py_test.go

```diff
@@ -78,3 +82,34 @@ func processEnv(t *testing.T, pid int) map[string]string {
 	}
 	return env
 }
+
+func genExe(t *testing.T, name string) {
+	file, err := os.Create(name)
+	require.NoError(t, err)
+	file.Close()
+	err = os.Chmod(name, 0o766)
+	require.NoError(t, err)
+}
+
+func Test_findPython(t *testing.T) {
```

Comment:
just curious. If test_findPython failed, could we skip other tests, e.g. venv and run?

Reply:
That's what's `skipIfNoPython`  for